readme.md

```diff
@@ -627,7 +628,6 @@ Make sure to see [contributing.md](/contributing.md) for instructions on contrib
 * [OX](o/OX.oz)
 * [Oxygene](o/Oxygene.pas)
 * [o:XML](o/o%CB%B8XML)
-* [Org-mode](o/Org-mode.org)
```

Comment:
Org-mode needs to be replaced.

Reply:
?
Org-mode already there. it duplicated